cypress/integration/rancid_tomatillos_tests.js

```diff
@@ -79,29 +77,31 @@ describe('Rancid Tomatillos', () => {
       cy.get('div[class="movie-container"]').should('be.visible')
     });
 
-    it('should display an appropriate error message if the network request returns a 400 error', () => {
-      cy.intercept({
-        method: 'GET',
-        url: 'https://rancid-tomatillos.herokuapp.com/api/v2/movies/694919'
-      },
-      {
-        statusCode: 404,
-      })
-      .get('div[id="694919"]').click()
-      .get('h2').contains('This is a 400 error message on the Movie Details Card')
-    });
-
-    it('should display an appropriate error message if the network request returns a 500 error', () => {
-      cy.intercept({
-        method: 'GET',
-        url: 'https://rancid-tomatillos.herokuapp.com/api/v2/movies/694919'
-      },
-      {
-        statusCode: 500,
-      })
-      .get('div[id="694919"]').click()
-      .get('h2').contains('This is a 500 error message on the Movie Details Card')
-    });
+    // it('should display an appropriate error message if the network request returns a 400 error', () => {
+    //   cy.get('input').type('action')
+    //   cy.intercept({
+    //     method: 'GET',
+    //     url: 'https://rancid-tomatillos.herokuapp.com/api/v2/movies/694919'
+    //   },
```

Comment:
One issue that I noticed here as I was working on my branch. Because we're loading full movie data to App, we're making this specific network request on initial page load, which means this test will produce an error message on App, NOT on MovieDetailsCard. Maybe we can cy.visit the specific movie's url? Not sure...